src/js/actions/transform.js

```diff
@@ -231,7 +235,40 @@ define(function (require, exports) {
         // Used to calculate the new top/left positions of all layers in relation to top/left
         // of the group
         var overallWidthRatio = size.w ? size.w / overallBounds.width : 0,
-            overallHeightRatio = size.h ? size.h / overallBounds.height : 0;
+            overallHeightRatio = size.h ? size.h / overallBounds.height : 0,
+            overallLeftOffset = 0,
```

Comment:
This may be on overreaction, but my gut wishes that all the character evaluation of the `refPoint.charAt()` should be stuffed away in a util file, along side its cousin `getPositionKeysByRefPoint()`.  TBH, I think this "decode weird photoshop data" stuff belongs in our adapter layer, but that's a post-oct discussion.